tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/store/debugger_reducers.ts

```diff
@@ -111,6 +111,8 @@ const initialState: DebuggerState = {
     lastLoadedTimeInMs: null,
   },
   activeRunId: null,
+  lastDataPollTime: -1,
+  lastNewPollDataTime: -1,
```

Comment:
`-1` value here will result in off by one arithmetic on the selector since you are doing `lastDataPollTime` and `lastNewPollDataTime`.

Reply:
es, I'm subtracting `lastDataPollTime` from `lastNewPollDataTime`, i.e., `lastDataPollTime - lastNewDataTime`. These initial values are slightly arbitrary, but they serve the following case well.
- Initially, before any polling happens, the difference is 0, which causes the polling interval to be the lower bound, i.e., what we want.

Added a comment here to explain that.